frontend/typescript/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    root: true,
+    extends: [
+        'airbnb-typescript',
```

Comment:
Shouldn't we extend our regular js front-end config, or at least copy its rules? 🤔

Reply:
Yup, it was there, don't know why it was deleted 🤦